pkgs/development/python-modules/transitions/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, six, nose, mock, dill, pycodestyle }:
+
+buildPythonPackage rec {
+  pname = "transitions";
+  version = "0.6.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ikxsjg7vil0yhiwhiimnjzcb1ig6g6g79sdhs9v8rnrszk1mi2n";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "dill<0.2.7" dill
```

Comment:
Would you mind pinging upstream about this?
This seems to have been introduced with https://github.com/pytransitions/transitions/commit/19ec2abbe2c786156962860a00ee74e41f6f06ef and since then forgotten?
Are we not seeing the same issues for some reason?

Reply:
No, I'm not seeing any errors. I've opened an upstream issue.

Reply:
This is no longer a problem according to https://github.com/pytransitions/transitions/issues/286#issuecomment-361069494.

Reply:
Alright, thanks! 👍